buildconfig/CMake/Packaging/launch_workbench.bat

```diff
@@ -0,0 +1,32 @@
+@echo off
```

Comment:
The batch script was in case PowerShell is missing - but I think we established that PowerShell 2.0 is guaranteed to be available on a fresh Win7 without updates, so I think the batch file can be removed completely, what do you think?

Reply:
I'm happy with that. It reduces the number of files that are named in a similar manner and should reduce confusion around what to use.